pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/SegmentCreationJob.java

```diff
@@ -108,15 +108,15 @@ protected boolean isDataFile(String fileName) {
       return true;
     }
     return fileName.endsWith(".avro") || fileName.endsWith(".csv") || fileName.endsWith(".json") || fileName
-        .endsWith(".thrift");
+        .endsWith(".thrift") || fileName.endsWith(".parquet") || fileName.endsWith(".orc");
```

Comment:
You shouldn't need fileName.endsWith(".orc"), as the way ORC readers are currently configured go through JobConfigConstants.RECORD_READER_PATH